forgerock-auth/src/main/java/org/forgerock/android/auth/Node.java

```diff
@@ -38,6 +33,15 @@ public class Node implements Serializable {
     private final String authServiceId;
     private final List<Callback> callbacks;
 
+    public Node(String authId, String stage, String header, String description, String authServiceId, List<Callback> callbacks) {
```

Comment:
Javadoc?

Reply:
Mark is as internal use only, does not expect developer to create the Node.